www/gatsby-node.js

```diff
@@ -268,7 +268,12 @@ exports.createPages = ({ graphql, actions, reporter }) => {
 
   createRedirect({
```

Comment:
I guess we need to add these redirects too?
```
createRedirect({
  fromPath: `/tutorial/advanced-tutorials/`,
  toPath: `/tutorial/additional-tutorials/`,
  isPermanent: true,
})
createRedirect({
  fromPath: `/tutorial/advanced-tutorials/`,
  toPath: `/tutorial/additional-tutorials/`,
  isPermanent: true,
})
```

Reply:
I can not see a difference between this 2 🤔

Reply:
I think @wardpeet meant 

createRedirect({
    fromPath: `/docs/advanced-tutorials/`,
    toPath: `/tutorial/additional-tutorials/`,
    isPermanent: true,	    isPermanent: true,
  })

createRedirect({
    fromPath: `/tutorial/advanced-tutorials/`,
    toPath: `/tutorial/additional-tutorials/`,
    isPermanent: true,	    isPermanent: true,
  })